pkg/volume/rbd/rbd_test.go

```diff
@@ -131,7 +131,7 @@ func TestCanSupport(t *testing.T) {
 	}
 }
 
-type fakeDiskManager struct {
+type FakeDiskManager struct {
```

Comment:
Why this type needs to be public?

Reply:
Sorry for the late reply. There is a golint problem with exported func `NewFakeDiskManager` that indicates it returning unexported type `*rbd.fakeDiskManager`. So I fixed this problem with exported it.

But after receiving your reminder, I found that the func `NewFakeDiskManager `is only used in this package. It is better to revoke the exportation of  func `NewFakeDiskManager` than export type `fakeDiskManager`?

Reply:
`newFakeDiskManager` /`fakeDiskManager` sounds better, IMO.

Reply:
Okay, I got it!  I will update it, thanks.

Reply:
Updated. PTAL. Thanks.

Reply:
Updated. Thanks.